Tests/PackageLoadingTests/ConventionTests.swift

```diff
@@ -840,6 +840,26 @@ class ConventionTests: XCTestCase {
         }
     }
 
+    func testInvalidManifestConfigForNonSystemModules() {
+        var fs = InMemoryFileSystem(emptyFiles:
+            "/Sources/main.swift"
+        )
+        var package = PackageDescription.Package(name: "pkg", pkgConfig: "foo")
+
+        PackageBuilderTester(package, in: fs) { result in
+            result.checkDiagnostic("invalid configuration in pkg: pkgConfig should only be used with a System Module Package")
+        }
+
+        fs = InMemoryFileSystem(emptyFiles:
+            "/Sources/Foo/main.c"
+        )
+        package = PackageDescription.Package(name: "pkg", providers: [.Brew("foo")])
+
+        PackageBuilderTester(package, in: fs) { result in
+            result.checkDiagnostic("invalid configuration in pkg: providers should only be used with a System Module Package")
```

Comment:
thats the package name in this test case
`invalid configuration in <package_name>: ...`
should I make it
`invalid configuration in package <package_name>: ...`